internal/service/ecs/task_set.go

```diff
@@ -0,0 +1,744 @@
+package ecs
```

Comment:
The diff between this and the current file is at https://gist.github.com/DavidMah/c7992d8fc651d1e323536300bd3a2b4a

Library functions are renamed, but no logic has been rearranged other than changing the callsites to those library functions